### DISCUSSION TITLE
What is this Google error about in the browser console?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#67921] [Help]
### Summary

Error:
![image](https://github.com/user-attachments/assets/fe759655-7a60-4ff9-b513-b6ce98a15ec5)

^^ This is for a new project with just a few files created.    Why would I be getting this error? And how could I make it go away? 
https://github.com/Elindo586/estuff/tree/main/app

**Comment by icyJoseph** [comment]
Can you see the same issue on an incognito browser window?

**Reply by Elindo586** [reply]
Oh look... it isn't in incognito browser window.. why wouldn't it be there?

I do another note instead:

![image](https://github.com/user-attachments/assets/344eba99-3404-4093-a6c3-a4b9d86603d1)

**Reply by icyJoseph** [reply]
The last screenshot is because the React DevTools are not installed, because they are a Browser Extension.

In fact, going back to the original issue. The problem is most likely a Browser Extension you have installed in your Browser. Often some mess with the HTML and can cause these - issues. A future version of React will fix it. 

It ought to be a browser extension that prevents Google Analytics somehow?
